lighthouse-core/lib/network-recorder.js

```diff
@@ -189,64 +191,86 @@ class NetworkRecorder extends EventEmitter {
    * @param {LH.Crdp.Network.RequestWillBeSentEvent} data
    */
   onRequestWillBeSent(data) {
-    // NOTE: data.timestamp -> time, data.type -> resourceType
-    this.networkManager._dispatcher.requestWillBeSent(data.requestId,
-        data.frameId, data.loaderId, data.documentURL, data.request,
-        data.timestamp, data.wallTime, data.initiator, data.redirectResponse,
-        data.type);
+    const originalRequest = this._findRealRequest(data.requestId);
+    if (originalRequest) {
```

Comment:
a comment on what this case is would be 👍👍

Reply:
done